docs/UserGuide.md

```diff
@@ -122,14 +124,24 @@ The app helps students to consolidate important data, such as profiles of profes
     </tabs>
 </box>
 
-4. The application should launch, resembling the UI shown in the Home page.
+4. The application should launch, resembling the UI shown in the [UI mockup](#ui-mockup) below
```

Comment:
since u deleted the "UI Mockup" section then there is no need for the hyperlink, can just do `resembling the UI shown  below`